heartbeat/nodejs/heartbeat.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+endtime=$(date -u -v+1d +%s)
+
+while [[ $(date -u +%s) -le $endtime ]]
+do
+    time=`date +%H:%M:%S`
+    echo "Time Now: $time"
+    data=`printf '{ 
+            "commands": [{
+                "method": "upsert",
+                "collection": "heartbeat",
+                "id": "heartbeat",
+                "value": {
+                    "time": "%s"
+                }
+            }]
+        }' $time`
+
+    echo $data
+    curl --location --request POST 'APP_ID.cloud.ditto.live/api/v2/store/write' \
```

Comment:
We may want to adopt the v3 API since the end-of-life for v2 is Jan 18.

Reply:
Good point